apps/test/unit/templates/progress/progressHelpersTest.js

```diff
@@ -186,6 +187,22 @@ describe('progressHelpers', () => {
     });
   });
 
+  describe('isLevelAssessment', () => {
+    it('returns true if level kind is assessment', () => {
```

Comment:
question: how will the multi-page levels previously known as assessments be handled by this -- is the expectation that those levels are all `LevelKind.assessment `?

Reply:
Each of the levels in a level group has their kind marked as assessment from what I saw in testing.